Makefile

```diff
@@ -160,6 +160,11 @@ dockerized-test: ## Run all the dockerized tests
 		-e ES_USER=$${ES_USER} \
 		-e ES_PASS=$${ES_PASS} \
 		-e ELASTIC_APM_SECRET_TOKEN=$${ELASTIC_APM_SECRET_TOKEN} \
+		-e OTEL_EXPORTER_OTLP_ENDPOINT \
+		-e OTEL_EXPORTER_OTLP_HEADERS \
+		-e OTEL_SERVICE_NAME \
```

Comment:
IIRC, this service name is not defined in the `withOtelEnv`

- See https://github.com/elastic/apm-pipeline-library/tree/master/vars#withotelenv

The ones defined by the OTEL plugin are:

- https://github.com/jenkinsci/opentelemetry-plugin/blob/d3a89fd1e3f87103cc140499262fd8e42df8b283/src/main/java/io/jenkins/plugins/opentelemetry/JenkinsOpenTelemetryPluginConfiguration.java#L233-L254

I'll probably raise a PR in the apm-pipeline-library to set those env variable if version < 0.19

Reply:
```suggestion
		-e OTEL_SERVICE_NAME="apm-integration-testing" \
```